library/modules/Textures.cpp

```diff
@@ -142,6 +165,34 @@ long Textures::getTexposByHandle(TexposHandle handle) {
     return -1;
 }
 
+TexposHandle Textures::createTile(std::vector<uint32_t>& pixels, int tile_px_w, int tile_px_h) {
+    auto texture = create_texture(pixels, tile_px_w, tile_px_h);
+    auto handle = Textures::loadTexture(texture);
+    return handle;
+}
+
+std::vector<TexposHandle> Textures::createTileset(std::vector<uint32_t>& pixels, int texture_px_w,
+                                                  int texture_px_h, int tile_px_w, int tile_px_h) {
+    auto texture = create_texture(pixels, texture_px_w, texture_px_h);
```

Comment:
what happens to this surface after it is sliced? does it need to be freed?

Reply:
It does in `slice_tileset` after creation of separate tiles.